src/models/Happ.js

```diff
@@ -0,0 +1,11 @@
+export const defaultHapp = {
```

Comment:
I'm just using files in the models folder to collect any code that knows specific details about specific graphql types. So this refers to the Happ type in graphql, and similarly there is one for Transaction. We can just expand as needed.